src/sniffer.c

```diff
@@ -6258,6 +6259,10 @@ static int ProcessMessage(const byte* sslFrame, SnifferSession* session,
             sslFrame += rhSize;
             sslBytes -= rhSize;
             break;
+#ifdef WOLFSSL_DTLS13
+        case ack:
+            /* TODO */
```

Comment:
I didn't exactly know what to do here so I put todo... Is fallthrough alright?  If so, I can change to fallthrough. 
Let me know please.

Reply:
Yes this is fine for the sniffer. Thanks